1.8/administration/theme-problems.md

```diff
@@ -0,0 +1,14 @@
+# Broken themes
+If your theme doesn't look right or is completely broken you can try the following things to help you fix it.
+
+## Check your Board URL
+If your board URL is not set correctly then the paths to the resources in your theme will be broken. To check your board URL go to *ACP > Configuration > General Configuration* > edit the *Board URL* field. Make sure you include **http://** in your URL and don't put a trailing slash.
+
+## Deleted cache file
+The CSS of your theme is cached in the cache directory and if this is modified or deleted (for example by accident during an upgrade) you can regenerate it by saving your theme again in the theme editor. To do this go to *ACP > Templates & Styles > Select theme > global.css > Save changes*.
```

Comment:
This should be:
The CSS of your theme is cached in the cache directory and if this is modified or deleted (for example by accident during an upgrade) you can regenerate it by saving your theme again in the theme editor. To do this go to *ACP > Templates & Style > Select theme > global.css > Save changes*.